src/main/kotlin/com/quipalup/katydid/logentry/domain/LogEntryError.kt

```diff
@@ -14,3 +14,8 @@ sealed class DeleteLogEntryError {
     object DoesNotExist : DeleteLogEntryError()
     object Unknown : DeleteLogEntryError()
 }
+
+sealed class UpdateLogEntryError {
```

Comment:
I suggest to refactor this and object ` DoesNotExist : UpdateLogEntryError()` to `SaveLogEntryError`, since an error for this operation won't happen
https://www.loom.com/share/492c6e791f6b40e4ac255679d04e7434